File: UserStory/UserStory/Models/ModelUserStory/UserStoryDTO.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using UserStory.Models.ModelPP;
using UserStory.Models.OtherServices;

namespace UserStory.Models.ModelUserStory
{
    public class UserStoryDTO
    {
        public Guid UserStoryRootId { get; set; }
        public string TextUserStory { get; set; }
        public string PartOfEpic { get; set; }
        public Guid PrioritetizationParameterId { get; set; }
        public PrioritetizationParameter PrioritetizationParameter { get; set; }

        public Guid BacklogId { get; set; }
        public Guid? SprintId { get; set; }
    }
}

File: FrontendTemplateProject-main/src/app/models/UserStoryModel/userStoryDto.ts
import { Guid } from 'guid-typescript';

export class UserStoryRootDto {
    
    userStoryRootId!: Guid;
    textUserStory!: string;
    partOfEpic!: string;
    prioritetizationParameterId!: Guid;
    
    constructor(
        userStoryRootId: Guid,
        textUserStory: string,
        partOfEpic: string,
        prioritetizationParameterId: Guid
    ) {
        this.userStoryRootId = userStoryRootId;
        this.textUserStory = textUserStory;
        this.partOfEpic = partOfEpic;
        this.prioritetizationParameterId = prioritetizationParameterId;
    }
    
}